www/main/viewer/utils/chromecast.js

```diff
@@ -61,13 +61,16 @@ const getSanitizedViewer = () => {
   const viewerHtml = document.querySelector('#viewer-container')
     ? document.querySelector('#viewer-container').cloneNode(true)
     : '';
-  viewerHtml.querySelector('.viewer-logo').remove();
-  viewerHtml.querySelector('.slide-quicktools').remove();
-  if (viewerHtml.querySelector('video')) {
-    viewerHtml.querySelector('video').remove();
-  }
-  viewerHtml.querySelector('.shabad-deck').removeAttribute('style');
-  viewerHtml.querySelector('.verse-slide-wrapper').removeAttribute('style');
+  viewerHtml.querySelector('.viewer-logo')?.remove();
+
+  viewerHtml.querySelector('.slide-quicktools')?.remove();
+
+  viewerHtml.querySelector('video')?.remove();
+
+  viewerHtml.querySelector('.shabad-deck')?.removeAttribute('style');
+
+ viewerHtml.querySelector('.verse-slide-wrapper')?.removeAttribute('style');
```

Comment:
Not a blocker. We don't need extra lines in-between, readability is fine without those. 
```suggestion
  viewerHtml.querySelector('.viewer-logo')?.remove();
  viewerHtml.querySelector('.slide-quicktools')?.remove();
  viewerHtml.querySelector('video')?.remove();
  viewerHtml.querySelector('.shabad-deck')?.removeAttribute('style');
 viewerHtml.querySelector('.verse-slide-wrapper')?.removeAttribute('style');
```